### DISCUSSION TITLE
Tutorisal...

### DISCUSSION THREAD

**Discussion by kenshin7311** [GSDragoon/RadeonSoftwareSlimmer#110] [General]
Has anyone made a tutorial on debloating and installing this.. the instructions arent very concise to say the leasts very least, I am getting Adrenalin crashes constantly

**Comment by uDEV2019** [comment]
Do you mean what of the drivers/components you should disable and/or not disable?

I'm remove everything that i don't need on my RX6800, such as Soundwire.
Screenshots from the three parts i've made (based on 24.2.1):
![RadeonSlimmer_24 2 1_01](https://github.com/GSDragoon/RadeonSoftwareSlimmer/assets/51911097/195f6425-d200-4e12-b245-86a04f879b5c)
![RadeonSlimmer_24 2 1_02](https://github.com/GSDragoon/RadeonSoftwareSlimmer/assets/51911097/b10ce03e-390a-4bf3-85f5-a706064cad36)
![RadeonSlimmer_24 2 1_03](https://github.com/GSDragoon/RadeonSoftwareSlimmer/assets/51911097/6266671a-ea08-4cfe-9788-432cfc5c14ad)

With this i don't have any crashes of Adrenalin.
But i don't think that your crashes are not involving some third-party application/driver.
